src/schema/v2/OrderedSet/orderedSetsConnection.ts

```diff
@@ -0,0 +1,61 @@
+import { GraphQLFieldConfig, GraphQLString } from "graphql"
+import { ResolverContext } from "types/graphql"
+import { paginationResolver } from "schema/v2/fields/pagination"
+import { pageable } from "relay-cursor-paging"
+import { convertConnectionArgsToGravityArgs } from "lib/helpers"
+import { OrderedSetConnection } from "./OrderedSet"
+
+export const OrderedSetsConnection: GraphQLFieldConfig<
```

Comment:
It's not clear if we have a strong naming preference, but I included `connection` at the end of this new field to differentiate it from the existing `OrderedSets` [field](https://github.com/artsy/metaphysics/blob/1ab7e66e31eb19e9e8a28e21df079b78934eb68a/src/schema/v2/OrderedSet/orderedSets.ts#L11).

Reply:
I think we do have a preference to append `connection` as a suffix! From https://github.com/artsy/README/blob/main/playbooks/graphql-schema-design.md#connections-over-lists

Reply:
Question, I feel like the phrase 'ordered' doesn't add a whole lot here. Maybe the root field should be called `setsConnection`?

Reply:
Definitely agree with you, but think it falls into the same category as this [comment](https://github.com/artsy/metaphysics/pull/4599#discussion_r1050031042) (i.e. making a trade-off to be consistent with established naming)